src/App.js

```diff
@@ -2,14 +2,17 @@ import { BrowserRouter as Router, Routes, Route } from "react-router-dom";
 import "./App.css";
 import Login from "./pages/login";
 import Home from "./pages/home";
+import Card from "./pages/card";
 
 function App() {
   return (
     <div className="App">
+      <Card />
       <Router>
         <Routes>
           <Route path="login" element={<Login />} />
           <Route path="home" element={<Home />} />
+          <Route path="card" element={<Card />} />
```

Comment:
Since `Card` is a re-usable component, it does not need it's own Route and we can remove this. I added the Readme of this repo to show the mockup.